CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.1.1 (2021-03-18)
```

Comment:
I think you also need the following commits:
- https://github.com/Azure/iotedge/pull/4589
- https://github.com/Azure/iotedge/pull/4451
 -https://github.com/Azure/iotedge/pull/4444

Reply:
#4444 was included in #4529 - the previous merge.